test/extensions/filters/http/cache/http_cache_utils_test.cc

```diff
@@ -0,0 +1,81 @@
+#include <vector>
+
+#include "common/http/header_map_impl.h"
+
+#include "extensions/filters/http/cache/http_cache_utils.h"
+
+#include "test/test_common/utility.h"
+
+#include "gtest/gtest.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace HttpFilters {
+namespace Cache {
+namespace {
+
+// TODO(toddmgreer): Add tests for eat* functions
+// TODO(toddmgreer): More tests for httpTime, effectiveMaxAge
```

Comment:
Can you make sure this is covered by issues? Can you also open an issue on fuzzing for the cache filter components? All of these definitely need fuzzing.

Reply:
done